blueprint/src/sections/hellinger_alpha.tex

```diff
@@ -22,7 +22,7 @@ \chapter{Hellinger alpha-divergences}
   \lean{ProbabilityTheory.hellingerDiv_ne_top_of_le_one}
   \leanok
   \uses{def:hellingerAlpha}
-  For $\alpha \in (0, 1)$ and finite measures $\mu, \nu$, $\He_\alpha(\mu, \nu) < \infty$.
+  For $\alpha \in (0, 1]$ and finite measures $\mu, \nu$, $\He_\alpha(\mu, \nu) < \infty$.
```

Comment:
The right definition of the Hellinger divergence at 1 is not the f-divergence with the constant 0 function (as we get in the code), but KL, as written in the blueprint. So this is not true at alpha = 1 and you should revert that change.
```suggestion
  For $\alpha \in (0, 1)$ and finite measures $\mu, \nu$, $\He_\alpha(\mu, \nu) < \infty$.
```